Porting/vote_admin_guide.pod

```diff
@@ -0,0 +1,176 @@
+=encoding utf8
+
+=head1 NAME
+
+vote_admin_guide - Perl Governance Vote Administrator Guide
+
+=head1 Running a Steering Council nomination process
+
+Announce the nomination period to the Perl Core mailing list.  Be explicit
+about the end time.  Set a calendar reminder for yourself for when it's
+time to close the nominations and begin the voting.
+
+If someone outside the Core Team is nominated, contact them to confirm that
+they accept the nomination.  If they do, ask the Steering Council to add
+them as a moderated member of the Perl Core list for the duration of the
+election and invite them to offer a statement on their candidacy to the
+list.
+
+Remind inactive Core Team members that they have until the end of
+nominations to reactivate themselves if they wish to vote.
```

Comment:
perlgov says this:
> Core Team members who have stopped contributing are encouraged to declare themselves "inactive". Inactive members do not nominate or vote.  Inactive members may declare themselves active at any time, except when a vote has been proposed and is not concluded.  Eligibility to nominate or vote will be determined by the Vote Administrator.

When is a vote "proposed" in this case?  I would have thought that when nominations begin, the vote is proposed, and an inactive member must wait it out.

Reply:
That's a good point and we've not been doing that.  I'll update this.